coalib/output/ConfWriter.py

```diff
@@ -105,7 +105,7 @@ def __write_key_val(self, keys, val):
 
         self.__file.write((self.__key_delimiter + " ").join(keys) + " " +
                           self.__key_value_delimiter + " " + val + "\n")
-        self.__wrote_newline = False
+        self.__wrote_newline = True
```

Comment:
If we do that then when a default section does not exist in a .coafile then the file would be
`[Default]
[some_section]` is that ok? They would not be any new line after `[Default]`